Tests/ArgumentParserUnitTests/ParsableArgumentsValidationTests.swift

```diff
@@ -106,11 +104,48 @@ final class CodingKeyValidationTests: XCTestCase {
     XCTAssertThrowsError(try ParsableArgumentsCodingKeyValidator.validate(E.self)) { (error) in
       if let error = error as? ParsableArgumentsCodingKeyValidator.Error {
         XCTAssert(error.missingCodingKeys == ["phrase", "includeCounter"])
-        XCTAssert(error.parsableArgumentsType.init() is E)
       } else {
         XCTFail()
       }
     }
   }
   
+  private struct F: ParsableArguments {
+    @Argument()
+    var phrase: String
+    
+    @Argument()
+    var items: [Int]
+  }
+  
+  private struct G: ParsableArguments {
+    @Argument()
+    var items: [Int]
+    
+    @Argument()
+    var phrase: String
+  }
+  
+  private struct H: ParsableArguments {
+    @Argument()
+    var items: [Int]
+    
+    @Option()
+    var option: Bool
+  }
+  
+  func testPositionalArgumentsValidation() throws {
+    try PositionalArgumentsValidator.validate(A.self)
+    try PositionalArgumentsValidator.validate(F.self)
+    XCTAssertThrowsError(try PositionalArgumentsValidator.validate(G.self)) { error in
+      if let error = error as? PositionalArgumentsValidator.Error {
+        XCTAssert(error.positionalArgumentFollowingRepeated == "phrase")
+        XCTAssert(error.repeatedPositionalArgument == "items")
+      } else {
+        XCTFail()
+      }
+      XCTAssert(error is PositionalArgumentsValidator.Error)
+    }
+    try PositionalArgumentsValidator.validate(H.self)
+  }
```

Comment:
Could you add a test where the valid/invalid cases are reached by combining `@OptionGroup`s in different orders?